acme_diags/driver/streamflow_driver.py

```diff
@@ -303,9 +333,15 @@ def run_diag(parameter):
                     monthly = numpy.ones((12, 1))
                 else:
                     monthly = mmat
+
+                if type(monthly) == cdms2.tvariable.TransientVariable:
```

Comment:
(From https://github.com/pyutils/line_profiler) Using `kernprof -l demo_run.py` and `python -m line_profiler demo_run.py.lprof`, I found that the lines above take a while to run:
```
ref = numpy.squeeze(ref_array[ref_lon-1, ref_lat-1, :])
mmat = numpy.transpose(numpy.reshape(ref, (-1,12)))
monthly_mean_ref = numpy.nanmean(mmat, axis=1)
annual_mean_ref = numpy.mean(monthly_mean_ref)
```
They take 23.2, 3.2, 27.6, and 8.2 seconds respectively. Unfortunately, I couldn't move this `TransientVariable` check any further up because it would cause the Scatter Plot to have a `R^2` of 0 or take longer.